### PR TITLE
Change the initial letter of the log to uppercase in roles

### PR DESCRIPTION
**Pull request by daniel-hutao**
Change the initial letter of the log to uppercase in roles

### CODE REVIEW

File: README_zh.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 > [English](README.md) | 中文
 
-KubeSphere 支持在已有 Kubernetes 集群之上部署 [KubeSphere](https://kubesphere.io/)。
+KubeSphere 支持在已有 Kubernetes 集群之上部署 [KubeSphere](https://kubesphere.io/).
```
Comment (Sherlock113) [inline README_zh.md:5]: No need to change the period. The period is supposed to be a Chinese period as the file is in Chinese regardless of whether the last word of the sentence is Chinese or English.

File: README_zh.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ csi-qingcloud (default)   disk.csi.qingcloud.com    54d
 glusterfs                 kubernetes.io/glusterfs   3d4h
 ```
 
-4. CSR signing 功能在 kube-apiserver 中被激活，参考 [RKE installation issue](https://github.com/kubesphere/kubesphere/issues/1925#issuecomment-591698309)。
+4. CSR signing 功能在 kube-apiserver 中被激活，参考 [RKE installation issue](https://github.com/kubesphere/kubesphere/issues/1925#issuecomment-591698309).
```
Comment (Sherlock113) [inline README_zh.md:39]: Same here.

File: README_zh.md (hunk 1 above)
Comment (Sherlock113) [reply README_zh.md:5]: For example, this is Google's doc:

image

Comment (daniel-hutao) [reply README_zh.md:5]: You are right. It's no need to change 。to .      I just meet the 【 [xx](http://xxx)。】 cannot be parsed with Goland markdown plugin, 【[xx](http://xxx) 。】and 【[xx](http://xxx).】are both ok. How about change 【)。】to 【) 。】？Or do nothing there.

File: README_zh.md (hunk 2 above)
Comment (daniel-hutao) [reply README_zh.md:39]: Got it

**Comment by daniel-hutao** [comment]
https://github.com/kubesphere/ks-installer/pull/1237

**Pull request closed without merging**